testing/testcafe/tests/dataGrid/grouping/grouping.ts

```diff
@@ -259,3 +262,38 @@ safeSizeTest('Empty header message should appear when all columns grouped and se
     },
   });
 });
+
+test('Checking group panel and group rows via aXe', async (t) => {
+  const { error, results } = await axeCheck(t, undefined, {
+    rules: {
+      'color-contrast': { enabled: false },
+    },
+  });
+
+  await t
+    .expect(error)
+    .eql(null)
+    .expect(results.violations.length === 0)
+    .ok(createReport(results.violations));
```

Comment:
Too many lines for one check (that axe doesn't see any error), maybe move it to helper?